### PR TITLE
Implemented DC.toModel

### LINKED ISSUE #688: Implement DataController.toModel()
There's already `toView()`. The `toModel()` method will be super simple:

```js
toModel( viewFragmentOrElement, additionalData ) {
    return this.viewToModel( viewFragmentOrElement, additionalData );
}
```

but will make for a nice API parity.

### PR DESCRIPTION
**Pull request by Reinmar**
Fixes #688.

### CODE REVIEW

**Comment by Reinmar** [comment]
Besides adding the new method I also added tests for the other ones, to confirm that they work with document fragments. We used doc frag convertion already in the clipboard, so the engine code was outdated.

File: tests/controller/datacontroller.js [hunk 1]
```diff
@@ -293,26 +337,57 @@ describe( 'DataController', () => {
...
+		it( 'should convert a document fragment', () => {
+			const modelDocumentFragment = parseModel( '<paragraph>foo</paragraph><paragraph>bar</paragraph>', modelDocument.schema );
+
+			const viewDocumentFragment = data.toView( modelDocumentFragment );
+
+			expect( viewDocumentFragment ).to.be.instanceOf( ViewDocumentFragment );
+			expect( viewDocumentFragment ).to.have.property( 'childCount', 2 );
 
-			const viewElement = data.toView( modelElement ).getChild( 0 );
+			const viewElement = viewDocumentFragment.getChild( 0 );
```
Comment (scofalik) [inline tests/controller/datacontroller.js]: It would be more complete if you check both children.

Comment (Reinmar) [inline tests/controller/datacontroller.js]: Agree. But this is not a test for conversion itself, but for methods utilising conversion.

Comment (Reinmar) [inline tests/controller/datacontroller.js]: I mean – there are a lot more things which could be broken below, but it'd be unwise to try to test them so late. I'm checking the number of children and the first child to verify that enough elements were converted and that they make some sense.

**Pull request merged**